docker/Dockerfile

```diff
@@ -12,7 +12,7 @@ ENV WSGI_CONTAINER ${WSGI_CONTAINER:-gunicorn}
 COPY whls/*.whl /whls/
 
 RUN addgroup -S ovd \
- && adduser -S ovd -G ovd -u 100 \
+ && adduser -S ovd -G ovd -u 101 \
```

Comment:
is this change to 101 required for the uplift?

Reply:
Yeah for 100 is not allowed in the newer version of the alpine base image